docs/use_cases/retrieval_augmented_generation_eval_qdrant_ragas.md

```diff
@@ -0,0 +1,618 @@
+
+As [summarised](https://superlinked.com/vectorhub/evaluating-retrieval-augmented-generation-a-framework-for-assessment%23S1bk_&sa=D&source=editors&ust=1712248322105041&usg=AOvVaw2HS8eHu-8FLtIIF6qwFB7e) in the first article:
+
+*An effective framework is essential to evaluate similarity and content overlap between generated response and ground-truths. To facilitate the same the first step is to create an evaluation dataset with questions , answers / ground truth with relevant context.*
+
+Also , a lot has happened since the first article , explicitly, the debate about if RAG is [still valuable](https://qdrant.tech/articles/rag-is-dead) when we have long-context LLMs (Large Language Models) like Gemini.
+
+While Large Language Models (LLMs) exhibit impressive memory capacities, the incorporation of Retrieval Augmented Generation (RAG) introduces a specialised framework for tracing data lineage, a crucial facet in various scenarios.
+
+The extensive context windows of LLMs could potentially synergize with RAGs, bolstering their aptitude for handling multifaceted tasks or addressing concurrent challenges. Beyond mere information retrieval, RAGs encompass a spectrum of functionalities including ranking, privacy management, and access control.
+
+However, LLMs equipped with elongated context windows may grapple with focusing on pertinent details amidst expansive information repositories, notwithstanding their comprehensive scope facilitating thorough data analysis.
+
+RAGs play a particularly pivotal role in enterprise applications, offering essential features such as control, auditability, and security that may not be fully addressed through direct interactions with LLMs alone.
+
+Now that we have reemphasised the importance and relevance of RAG, let's look at what this article plans to cover.
+
+1) **Synthetic Evaluation Dataset**<br>
+   a) Significance of evaluation dataset<br>
+   b. Different way to build it<br>
+   c. FAQs and Suggestions<br>
+2) **Metrics**<br>
+   a. Different metrics provided by the framework<br>
+   b. Choosing the right metrics for your use cases<br>
+   c. Learning to decode the metric values<br>
+3) **Learning to use evaluation dataset to evaluate given metrics**<br>
+4) **Code walkthrough and reference notebook**<br>
+
+>  💡 In this article, we utilize two Hugging Face datasets: [qdrant_doc](https://huggingface.co/datasets/atitaarora/qdrant_doc) and [qdrant_doc_qna](https://huggingface.co/datasets/atitaarora/qdrant_doc_qna). The dataset [qdrant_doc](https://huggingface.co/datasets/atitaarora/qdrant_doc) serves as the reference dataset upon which our Retrieval-Augmented Generation (RAG) solution is built. On the other hand, the dataset [qdrant_doc_qna](https://huggingface.co/datasets/atitaarora/qdrant_doc_qna) is employed as the golden-set for evaluation purposes. Keep this distinction in mind as you follow along with our analysis and results.
+
+
+### **Synthetic Evaluation Dataset**
```

Comment:
I think why I have to generate a dataset for evaluation is there between the lines, but it should be more explicit.